src/renderer/helpers/strings.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * This will return true if a string is null, undefined or empty.
+ * @param {string|null|undefined} _string the string to process
+ * @returns {boolean} whether the string is empty or not
+ */
+export function isNullOrEmpty(_string) {
+  return _string == null || _string === ''
+}
+
+/**
+ * Is KeyboardEvent.key a printable char
+ * @param {string} eventKey the string from KeyboardEvent.key to process
+ * @returns {boolean} whether the string from KeyboardEvent.key is a printable char or not
+ */
+export function IsKeyboardEventKeyPrintableChar(eventKey) {
```

Comment:
Can you please changing the naming of this function, to align it with the naming scheme used for other functions, starting with a lowercase letter. (Have you been writing some C# code recently 😅)

Reply:
It was `keyboardEventKeyIsPrintableChar`
Then I forgot to change the first char